src/internet_identity/src/storage.rs

```diff
@@ -375,6 +420,38 @@ impl<M: Memory + Clone> Storage<M> {
         self.flush();
     }
 
+    /// Add a new archive entry to the buffer.
+    pub fn add_archive_entry(&mut self, entry: BufferedEntry) {
+        self.archive_entries_buffer
```

Comment:
would it make sense to (double) check here, and in the new helpers below, that we are indeed using v8?

Reply:
Yes, that makes sense. Thanks!